src/main/java/de/tum/in/www1/artemis/security/jwt/TokenProvider.java

```diff
@@ -74,6 +76,22 @@ public String createToken(Authentication authentication, boolean rememberMe) {
         return Jwts.builder().setSubject(authentication.getName()).claim(AUTHORITIES_KEY, authorities).signWith(key, SignatureAlgorithm.HS512).setExpiration(validity).compact();
     }
 
+    /**
+     * Generates a access token that allows user to download a file. This token is only valid for 30 seconds.
```

Comment:
@maximilianmeier  "This token is only valid for 30 seconds." is a bit confusing because the duration is defined by the parameter durationValidityInSeconds and not set to 30s per default. Also it would be great if you could check if the JavaDoc does not miss any descriptions (e.g. here for @param/@result). It could cause a failing build if https://github.com/ls1intum/Artemis/pull/709 is merged.

Reply:
yes you are right thanks